frontend/debian/cuttlefish-orchestration.cuttlefish-host_orchestrator.default

```diff
@@ -24,6 +24,10 @@ orchestrator_https_port=2443
 # Defaults to "/tmp/<uid>/cvd_artifacts"
 orchestrator_cvd_artifacts_dir=/var/lib/cuttlefish-common
 #
+# Timeout for CVD creation
+# Defaults to 7 mins
+orchestrator_cvd_creation_timeout=7
```

Comment:
Should this line be uncommented? I think there's a default value in golang code.

Reply:
Changed to comment and also changed value of the variable.